overrides/config/ftbquests/quests/chapters/create.snbt

```diff
@@ -1,30 +1,3261 @@
+								lvl: 1
+							}]
+							HideFlags: 33
+							display: {
+								Lore: [
+									"{\"text\":\"§7§o§e§oContains a random special item!\"}"
+									"{\"text\":\"§b§7§o§bRight-Click to Open\"}"
+									"{\"text\":\"§5§6✎ §nLore Text\"}"
+									"{\"text\":\"§7§6§o§7§o\\\"An assortment of miscellaneous items of varying usefulness found within The Quest Master's storage locker and repackaged as your reward!\\\"\"}"
+									"{\"text\":\"\"}"
+								]
+								Name: "{\"text\":\"§eQuest Loot\"}"
+							}
+							"quark:RuneAttached": 1b
+							"quark:RuneColor": {
+								Count: 1b
+								id: "quark:rainbow_rune"
+							}
+							rpgloot.randomize: 0b
+							type: "quest_loot"
+						}
+					}
+					random_bonus: 2
+					type: "item"
+				}
+			]
+			size: 0.8d
+			subtitle: "One Needs to Eat to Think"
+			tasks: [
+				{
+					id: "4C615F418704E6BD"
+					item: "create:mechanical_bearing"
+					type: "item"
+				}
+				{
+					count: 3L
+					id: "5B40F85053578394"
+					item: "create:mechanical_harvester"
+					type: "item"
+				}
+				{
+					id: "0FE9F88C5D4A03E5"
+					item: "create:portable_storage_interface"
+					type: "item"
+				}
+				{
+					id: "25192AEC18268AB7"
+					item: "ratatouille:spreader"
+					type: "item"
+				}
+				{
+					id: "16ECBA5125CD45BB"
+					item: "create:nozzle"
+					type: "item"
+				}
+				{
+					consume_items: true
+					count: 32L
+					id: "5CA7299185BD7C73"
+					item: { Count: 32, id: "minecraft:wheat" }
+					type: "item"
+				}
+			]
+			title: "Food for Thought"
+			x: -5.0d
+			y: 1.0d
+		}
+		{
+			dependencies: ["40A64D59F1186585"]
+			id: "11834C20D8EBA235"
+			rewards: [{
+				amount: 50L
+				currency: "eternalcurrencies:coins"
+				id: "7ACBB6295F9F1395"
+				ignore_reward_blocking: true
+				type: "eternalcurrencies:currency"
+			}]
+			shape: "diamond"
+			size: 0.65d
+			subtitle: "Essential in your Journey "
+			tasks: [{
+				count: 16L
+				id: "58608757A73381FC"
+				item: "create:iron_sheet"
+				type: "item"
+			}]
+			x: 1.33d
+			y: -2.57d
+		}
+		{
+			dependencies: ["40A64D59F1186585"]
+			id: "18DA2B3AE200EB17"
+			optional: true
+			rewards: [{
+				amount: 50L
+				currency: "eternalcurrencies:coins"
+				id: "08EDB15336E3C281"
+				ignore_reward_blocking: true
+				type: "eternalcurrencies:currency"
+			}]
+			shape: "diamond"
+			size: 0.65d
+			tasks: [{
+				count: 8L
+				id: "4C305F5BAB36E339"
+				item: "create:golden_sheet"
+				type: "item"
+			}]
+			x: 1.62d
+			y: -2.28d
+		}
+		{
+			dependencies: ["40A64D59F1186585"]
+			id: "731EB61AC3C46279"
+			optional: true
+			rewards: [{
+				amount: 50L
+				currency: "eternalcurrencies:coins"
+				id: "1FE5508C6DB5FC66"
+				ignore_reward_blocking: true
+				type: "eternalcurrencies:currency"
+			}]
+			shape: "diamond"
+			size: 0.65d
+			subtitle: "Great for Fluids"
+			tasks: [{
+				count: 8L
+				id: "7E74E99B876199D0"
+				item: "create:copper_sheet"
+				type: "item"

```

Comment:
i'd personally squish all of the Metal Sheet quests into one, and not make it require multiple of them, but that's just me. Sturdy Sheet being its own makes sense considering it's unlocked a bit later, and serves its own set of purposes.